CMake/fletch-tarballs.cmake

```diff
@@ -283,6 +283,12 @@ if(NOT WIN32)
   set(Caffe_md5 "84e39223115753b48312a8bf48c31f59")
   list(APPEND fletch_external_sources Caffe)
 endif()
+if(WIN32)
+  set(Caffe_version "527f97c0692f116ada7cb97eed8172ef7da05416")
+  set(Caffe_url "https://github.com/BVLC/caffe/archive/${Caffe_version}.zip")
+  set(Caffe_md5 "751e3cd16050e7ecd3981457e41b29cd")
+  list(APPEND fletch_external_sources Caffe)
+endif()
```

Comment:
I would add into a if-else block with the above as opposed to 2 opposite if statements for clarity